devices/Mhz19b/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CSharp.BlankApplication")]
```

Comment:
Please follow here the usual pattern with the copyright and other bits. (just check one of the existing bindings 😉  )
OK to remove the ones not used.

Reply:
Used the ahtxx device as an example.

Reply:
Got you. And yes, you've probably picked up one of the 1st ones to be ported where all this was being figured it out... 😅